models/stg_apple_store__app_session_daily.sql

```diff
@@ -0,0 +1,51 @@
+with base as (
+
+    select * 
+    from {{ ref('stg_apple_store__app_session_tmp') }}
+
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_apple_store__app_session_tmp')),
+                staging_columns=get_app_session_detailed_daily_columns()
```

Comment:
Is this staging_columns parameter pointing at the correct macro?

Reply:
Just double checked, yes I believe so. Are you asking because the name is different? I cut off some of the characters because some warehouses had issue with long file names.

Reply:
Thanks! Yeah it was tripping me up that the names were divergent. But this makes sense and I can see the same approach used for other models in this update. No change required here.